Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        static void Main(string[] args)
```

Comment:
You can only have one Main function. Look in the [bc-homework](https://discordapp.com/channels/475671343463923714/663436225255964697/668202071547183104) channel which file is supposed to have that Main function.